### PR TITLE
ci: add Fedora 37

### PR DESCRIPTION
**Pull request by ondrejbudai**
This commit needs to do two things:
- update the terraform definition
- add fedora-37 to the gitlab-ci file

I also removed the commented out aarch64 runners to clean the file a bit.

### CODE REVIEW

**Comment by gicmo** [comment]
```
osbuild 376cbffd136bc4ba86fc7c63697fa5b88fe3ace 2.5 kB/s | 303  B     00:00    
[2603](https://gitlab.com/redhat/services/products/image-builder/ci/koji-osbuild/-/jobs/3378678163#L2603)Errors during downloading metadata for repository 'osbuild':
[2604](https://gitlab.com/redhat/services/products/image-builder/ci/koji-osbuild/-/jobs/3378678163#L2604)  - Status code: 403 for http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com/osbuild/fedora-37/x86_64/376cbffd136bc4ba86fc7c63697fa5b88fe3acef/repodata/repomd.xml (IP: 52.219.176.72)
[2605](https://gitlab.com/redhat/services/products/image-builder/ci/koji-osbuild/-/jobs/3378678163#L2605)Error: Failed to download metadata for repo 'osbuild': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```

**Comment by ondrejbudai** [comment]
Ah, sure, we need newer versions of osbuild and composer. The old ones don't have F37 RPMs.

**Comment by thozza** [comment]
Do we actually need to keep testing on F36? 🤔

**Comment by ondrejbudai** [comment]
@thozza We release into both F36 and F37, so we probably should. Does it make sense?

**Comment by thozza** [comment]
> @thozza We release into both F36 and F37, so we probably should. Does it make sense?

fair...

File: schutzbot/deploy.sh [hunk 1]
```diff
@@ -20,8 +20,8 @@ function retry {
 
 # Variables for where to find osbuild-composer RPMs to test against
 DNF_REPO_BASEURL=http://osbuild-composer-repos.s3-website.us-east-2.amazonaws.com
-OSBUILD_COMMIT=376cbffd136bc4ba86fc7c63697fa5b88fe3acef             # release 62
-OSBUILD_COMPOSER_COMMIT=2d13050ba84a2bb30d9dd980639ee78354fe90f3    # release 59-dev (includes mock openid server fix)
+OSBUILD_COMMIT=fb5412931711be41fd19cdaf402f2ed89d1d60b8             # release 72
+OSBUILD_COMPOSER_COMMIT=587e0436681b74ecb359945066ae63afeba9085a    # release 68
```
Comment (thozza) [inline schutzbot/deploy.sh:24]: Any reason to not use the latest releases at this time?

**Review by thozza — DISMISSED** [review]
Looks good, but IMO it would make sense to update the PR to use the latest osbuild and composer releases...

File: schutzbot/deploy.sh (hunk 1 above)
Comment (ondrejbudai) [reply schutzbot/deploy.sh]: None, this PR is just old. Fixed in a force-push. ;)

**Review by thozza — DISMISSED** [review]
Thanks ❤️

**Comment by thozza** [comment]
@ondrejbudai integration test is failing on el8.6, specifically the upload to AWS.

There are no useful logs, no journal. 😢

**Comment by ondrejbudai** [comment]
I fixed it by increasing the apache timeout in the kojihub contianer. Apparently, we were hitting it while CGImporting the big raw image for AWS.

I'm pretty sure that this is happening in the CI for quite some time, but it was masked due to infamous https://github.com/osbuild/osbuild-composer/commit/972da81a91c0ff7d5360ceed66a051fc7aa9ce47

**Review by thozza — APPROVED** [review]
Thanks for your effort!

**Pull request merged**